HelpSource/Classes/Unpack1FFT.schelp

```diff
@@ -29,27 +29,28 @@ argument:: whichmeasure
 
 examples::
 code::
-(
-s.waitForBoot({
-	c = Buffer.read(s, Platform.resourceDir +/+ "sounds/a11wlk01.wav");
-})
-)
+
+s.waitForBoot { c = Buffer.read(s, Platform.resourceDir +/+ "sounds/a11wlk01.wav") }
```

Comment:
Wouldn't it be better to drop the `s.waitForBoot` and simply write `c = Buffer.read(...`  ? This example used to load two buffers. Now that we use only one, `waitForBoot` is not needed.

Reply:
I suppose, our assumption in the help files is anyhow that the server is booted. So yes, we can leave that out.

Whether there is one or two doesn't matter.